doc/setup.md

```diff
@@ -51,7 +51,7 @@ brew install geckodriver
 1. Install dependencies
 
 ```
-sudo apt install nginx supervisor postgresql libpq-dev nodejs
```

Comment:
I think removing nodejs here is not correct.  See https://github.com/skyportal/skyportal/blob/master/.github/workflows/test.yml#L80

Reply:
I'm not sure what the reference means, but I tested it on a new install of Ubuntu 20 and it works without specifying nodejs. 

I think what happens is that npm install it.

Reply:
I tested this line on a fresh install of Ubuntu 20, and everything works.

Reply:
@stefanv what do you say? can we merge this?